### PR TITLE
vm: Simplifying Deploy VM Wizard

### LINKED ISSUE #490: [BUG] Create VM Form - root disk size dialog is misleading
**Describe the bug**

The implementation of the root disk slider suggests that the user MUST set a root disk size. This has two problems:

1. This is not the case and setting a root disk size is conpletely optional and not always possible.
2. User will not know the original root disk size, and setting a root disk size smaller than the original disk will fail the deployment.

**To Reproduce**

1. Log in as: User
2. Go to 'Create Instance'
3. Select any zone
4. See Root disk slider

**Expected behavior**

A clear indication that this is optional is required.

**Screenshots**

image

**Desktop (please complete the following information):**
 - OS: [e.g. Windows, Mac, iOS, Android with device/screen details if applicable]

**Additional context**

### LINKED ISSUE #491: [BUG] The Create Instance wizard should have an 'Advanced section' to make requirements clear
**Describe the bug**

The Create Instance wizard should have a number of options moved to an optional 'Advanced' section, their existence in the 'main flow' is confusing even to experienced users, deployment would be near impossible for new users.

**To Reproduce**

 Example:  What should a user enter for 'root disk size'  (link to #490 )

**Expected behavior**

The following should be in a clearly optional advanced section.
'root disk size'
'Affinity groups'
'Boot type'
'boot into hardware setup'
'boot mode'
'keyboard language' (as redundant with novnc console)
'userdata'

**Screenshots**

**Desktop (please complete the following information):**
 - OS: [e.g. Windows, Mac, iOS, Android with device/screen details if applicable]

**Additional context**

### PR DESCRIPTION
**Pull request by davidjumani**
Fixes #490
Fixes #491 
 
- Moves Affinity Group selection to Details
- Sets the minimum of the root disk size sider to the template size
- Adds a switch to show / hide details
- Adds a switch to show / hide root disk size slider

Screenshot from 2020-07-06 19-29-36
Screenshot from 2020-07-06 19-29-47
Screenshot from 2020-07-06 19-44-39
Screenshot from 2020-07-06 19-44-54

### CODE REVIEW

**Comment by davidjumani** [comment]
@blueorangutan package

**Comment by davidjumani** [comment]
@PaulAngus Looks good ?

**Comment by davidjumani** [comment]
@blueorangutan package

**Comment by rhtyd** [comment]
@davidjumani on the override root disk side can you remove the slider; so just the input and are we checking that size input is great than the size of template?

**Comment by davidjumani** [comment]
@blueorangutan package

**Comment by rhtyd** [comment]
Boot type, etc options are gone @davidjumani 
The advanced step looks weird, instead how about we move it to the very end, still have the swtich to show/hide those options? What do you think @PaulAngus ?

@utchoang can you review and test this as well?

**Comment by davidjumani** [comment]
They aren't visible since we're using simulator. Tested it on a VMware setup @rhtyd 
Screenshot from 2020-07-07 18-45-23

**Comment by PaulAngus** [comment]
David demo'd it to me @rhtyd, I thought that it did the job.  I imagine there will be a lot of little tweaks we can make in future. but this iteration seemed a lot cleaner to me, so I gave it a thumbsup.

**Comment by rhtyd** [comment]
Okay we're doing hypervisor specific fields now, will test and merge.

**Comment by utchoang** [comment]
@blueorangutan package

File: src/views/compute/DeployVM.vue [hunk 1]
```diff
@@ -89,12 +89,18 @@
                           :selected="tabKey"
                           :loading="loading.templates"
                           :preFillContent="dataPreFill"
-                          @update-template-iso="updateFieldValue"
-                        ></template-iso-selection>
+                          @update-template-iso="updateFieldValue" />
+                        <span>
+                          {{ $t('Override root disk size') }}
```
Comment (utchoang) [inline src/views/compute/DeployVM.vue:94]: @davidjumani Can you fix translation/key label usage across views (see locales/en.json to pick the keys, all lowercase)

Comment (davidjumani) [reply src/views/compute/DeployVM.vue]: Done!

**Comment by davidjumani** [comment]
@blueorangutan package

**Pull request merged**